pkg/scripts/uninstall.sh

```diff
@@ -151,6 +151,15 @@ destroy_tfstate() {
   fi
 }
 
+cleanup_asg_ec2_tagging_module() {
+  echo "Cleaning up asg_ec2_tagging Module."
+  cd "${SCRIPT_PATH}/../modules/AWS/asg_ec2_tagging"
```

Comment:
If you change the current folder you need to make sure you back to the same folder when you done with cleaning. I mean your script should not change the current folder before the script.